tests/data/settings.json

```diff
@@ -228,6 +228,97 @@
     "version": 1
   },
 
+  "AB_T_100_W_25_25_25_25": {
+    "sdkKey": "someuniquestuff1234567",
+    "campaigns": [
+      {
+      "goals": [
+        {
+          "identifier": "CUSTOM",
+          "id": 213,
+          "type": "CUSTOM_GOAL"
+        }
+      ],
+      "variations": [
+        {
+          "id": 1,
+          "name": "Control",
+          "changes": {},
+          "weight": 25.0,
+          "isFeatureEnabled": false,
+          "variables": null,
+          "segments": {
+            "or": [
+              {
+                "user": " You, Faizan, Rohit "
```

Comment:
These names are not there in any other SDKs. Why added?

Reply:
To test a case where more that one variation passed whitelisting , would change these name s according to other sdks.

Reply:
This was one way of testing the case -> where more than one variation passed for whitelisting evaluation. other would have been to create E2E cases with custom_variable and variation_targeting_variables, But we aren't doing that in any sdk. Also i wasn't able to find test cases for this particular test in any other sdk.

Reply:
For now removed the Rohit and kept the original 26 user.